src/settings.cpp

```diff
@@ -53,6 +54,9 @@
 ClientSettings _settings_client;
 GameSettings _settings_game;     ///< Game settings of a running game or the scenario editor.
 GameSettings _settings_newgame;  ///< Game settings for new games (updated from the intro screen).
+BaseGraphics::Ini BaseGraphics::ini_data;
+std::string BaseSounds::ini_set;
+std::string BaseMusic::ini_set;
```

Comment:
Does it work to have these as static inline inside the class definitions (like how window hotkeys are set up), instead of dumped here?